libs/sdk/src/NativeRNBridge.ts

```
@@ -0,0 +1,52 @@
+  logLevel?: number;
+  autoSessionTracking?: boolean;
+  sessionTimeout?: number;
+  enableBackgroundMode?: boolean;
+  collectDeviceId?: boolean;
+  enableGzip?: boolean;
+  dbEncryption?: {
+    key: string;
+    enable: boolean;
+  };
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  withFactories?: Array<Record<string, unknown> | Function>;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)